packages/twenty-ui/src/theme/utils/getNextThemeColor.ts

```diff
@@ -1,9 +1,9 @@
 import { MAIN_COLOR_NAMES, ThemeColor } from '@ui/theme';
 
-export const getNextThemeColor = (currentColor: ThemeColor): ThemeColor => {
-  const currentColorIndex = MAIN_COLOR_NAMES.findIndex(
-    (color) => color === currentColor,
-  );
+export const getNextThemeColor = (currentColor?: ThemeColor): ThemeColor => {
+  const currentColorIndex = currentColor
+    ? MAIN_COLOR_NAMES.findIndex((color) => color === currentColor)
+    : -1;
```

Comment:
I feel putting to -1 is hacky!

In case currentColor is not defined, I would expect currentColorIndex to be undefined
and then const nextColorIndex = currentColorIndex ? (currentColorIndex + 1) % MAIN_COLOR_NAMES.length : MAIN_COLOR_NAMES[0]